### ISSUE TITLE
quota on smb share

### ISSUE THREAD

**Issue by eleaner** [lizardfs/lizardfs#773]
Hi,

I need to set up smb share on one of the lizard subfolders.
I limited the quota on this folder and even mounted it separately.
Still, df and samba share reports the complete size of the filesystem
Do I do something wrong or is it a bug?

**Comment by Zorlin** [comment]
No, as far as I'm aware it will always show the size of the FS, regardless
of quotas.

**Comment by eleaner** [comment]
I believe that moosefs can show the correct sizes. Would this be then a feature request?

**Comment by nickcoons** [comment]
The amount of free space shown is the amount of free space across the available drives in the chunkserver.  It can't really show it to you any other way, because it can't predict how goals will be set on yet non-existent files.

However, if you're wanting an SMB client to show a lesser amount of space, you can look into the "dfree command" option for Samba in the share declaration section of smb.conf.  Here, you can create a script that returns the information you want Samba to report to the client as total and available space.
